### ISSUE TITLE
Commentor action fails to add comment when PR is from a fork

### ISSUE THREAD

**Issue by matthew-chirgwin** [strimzi/strimzi-ui#60]
As a part of #35 a GitHub action was added to add a comment to a PR with details such as bundle sizes, test coverage etc. This works (when tested on your own fork), but fails on the Strimzi-ui repo with:

`HttpError: Resource not accessible by integration`

From some quick searching around, the error implies some permissions are missing, or this action will never work (as the request is from a fork) - https://github.com/actions/labeler/issues/12#issuecomment-549901347

This issue is to find out if this is fixable, and if not, alternatives to get the information currently not presented in the PR

**Comment by matthew-chirgwin** [comment]
https://github.com/actions/labeler/issues/12#issuecomment-704942502 summises what looks to be the required action needed - I don't think that is a safe/sensible option allowing someone who forks this repo the ability to change it via a workflow.

**Comment by matthew-chirgwin** [comment]
See also https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/ - `workflow_run` sounds interesting. Thoughts @JQrdan ?

**Comment by JQrdan** [comment]
What `pull_request_target` does is exactly the suggestion I made earlier to make this work. So that's great that it does exist! 

TL;DR of the article

`pull_request_target` runs workflows that are in the base branch (master in our case) instead of the incoming branch. This means that the workflow is trusted as it has already been approved and merged before it gets to run. Therefore, it has all the read/write permissions that we need.

**Comment by matthew-chirgwin** [comment]
> What `pull_request_target` does is exactly the suggestion I made earlier to make this work. So that's great that it does exist!
> 
> TL;DR of the article
> 
> `pull_request_target` runs workflows that are in the base branch (master in our case) instead of the incoming branch. This means that the workflow is trusted as it has already been approved and merged before it gets to run. Therefore, it has all the read/write permissions that we need.

Beg pardon, you are absolutley right - I interpreted it the other way around! Sounds like a quick win change :)
